pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -65,6 +66,46 @@ func expectedEBC(totCap, targetBC, recordedConcurrency, numPods float64) int32 {
 	return int32(math.Floor(totCap/targetUtilization*numPods - targetBC - recordedConcurrency))
 }
 
+func expectedExcessBCF(originalReadyPodsCount int, totalValue, observedStableValue, targetBurstCapacity float64) float64 {
```

Comment:
this is the same as the method above just with different argument names and order

Reply:
I have combined these two tests and added PanicM.

Reply:
I meant `expectedEBC` is an existing method. This method you wrote is a duplicate and can be removed.